feature-libs/cart/quick-order/components/quick-order/table/item/quick-order-item.component.html

```diff
@@ -9,8 +9,9 @@
         >
           <cx-media
             [container]="entry?.product?.images?.PRIMARY"
-            [alt]="entry.product.summary ?? entry.product.name"
+            [alt]="entry?.product?.name"
             format="cartIcon"
+            role="presentation"
```

Comment:
Seems the `alt` attribute is clean and works well now, so please remove `role="presentation"` as both cannot coexist. Their purposes contradict each other.

Reply:
Removed